docs/user-guide/01-blueprint-reference.md

```diff
@@ -46,12 +73,26 @@ version = "*"
 name = "openssh-server"
 version = "*"
 ```
+</TabItem>
+<TabItem value="hosted">
+```json
+{
+  "image_name": "tmux",
+  "distribution": "fedora-38",
+  "customizations": {
+    "packages": ["tmux", "openssh-server"]
+  }
+}
+```
+</TabItem>
+</Tabs>
 
 ## Content
 
 The content section determines what goes into the image from other sources such as packages, package groups, or containers. Content is defined at the root of the blueprint.
 
-- [Packages](#packages).
+- [Packages](#packages).  
+  <Highlight colorVar="hosted">[customizations.packages[]](https://developers.redhat.com/api-catalog/api/image-builder#schema-Customizations)</Highlight>
```

Comment:
I wonder about two things:
1. Should we drop the content listing? Now that we have docusaurus, we actually have a sidebar with navigation for sub-sections.
2. I would find it more helpful if the links were in the corresponding sections - at least in addition. Now when you click on a sidebar-navigation item to go to a customization, you don't see if it's on premise or service.
3. Maybe we should still spell out what is `on-premise` in each customization.  We have at least one (architecture) that we currently only support in the hosted service and it's not documented yet.

Reply:
> I wonder about two things:
> 
>     1. Should we drop the content listing? Now that we have docusaurus, we actually have a sidebar with navigation for sub-sections.

but the docusaurus sidebar on the left, does not include the Chapters of a page. Should we change that? 🤔

>     2. I would find it more helpful if the links were in the corresponding sections - at least in addition. Now when you click on a sidebar-navigation item to go to a customization, you don't see if it's on premise or service.
good idea, I'll change that.

>     3. Maybe we should still spell out what is `on-premise` in each customization.  We have at least one (architecture) that we currently only support in the hosted service and it's not documented yet.
I'll check

Reply:
There's a secondary sidebar with in-page navigation on the right :)
MOAR SIDEBARS!

![image](https://github.com/osbuild/osbuild.github.io/assets/261436/2d5285fb-e420-48cf-93df-371ca4f524d0)

Reply:
I think I'd actually prefer it if the sidebar that is currently on the right were also on the left so that you can see the levels of navigation you're in more clearly... But not sure if it's a standard feature and whether it's worth getting into the CSS.
(Also, not sure if everyone would agree)